packages/opentelemetry-sdk-node/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@opentelemetry/sdk-node",
+  "version": "0.8.3",
+  "description": "OpenTelemetry SDK for Node.js",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "build": "npm run compile",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
```

Comment:
Are we including markdown lint scripts in all packages now? If so they should be added.

Reply:
markdown lint is performed entirely from the root